### PR TITLE
feat(l10n): localize device list

### PR DESCRIPTION
**Pull request by vladikoff**
~~TODO: needs tests~~

waiting on https://github.com/mozilla/fxa-shared/pull/3

### CODE REVIEW

File: lib/routes/account.js [hunk 1]
```diff
@@ -1007,6 +1013,9 @@ module.exports = function (
               device.isCurrentDevice =
                 device.sessionToken.toString('hex') === sessionToken.tokenId.toString('hex')
 
+              device.lastAccessTimeFormatted = localizeTimestamp.format(device.lastAccessTime,
```
Comment (vladikoff) [inline lib/routes/account.js]: @vbudhram format used here

**Comment by vladikoff** [comment]
@vbudhram if you apply this patch: https://github.com/mozilla/fxa-content-server/pull/4033 it should give you:

 

the new formatted date

**Comment by vladikoff** [comment]
@vbudhram ok tests  passing now

**Comment by vbudhram** [comment]
screen shot 2016-08-17 at 10 10 37 am

LGTM 👍, merge at will

**Pull request merged**